lmnet/tests/lmnet_tests/networks_tests/test_quantized_network_arguments.py

```diff
@@ -0,0 +1,45 @@
+import tensorflow as tf
+
+from lmnet.networks.classification.lmnet_v0 import LmnetV0Quantize
+from lmnet.networks.classification.lmnet_v1 import LmnetV1Quantize
+from lmnet.networks.classification.darknet import DarknetQuantize
+from lmnet.networks.object_detection.lm_fyolo import LMFYoloQuantize
+from lmnet.networks.object_detection.yolo_v2_quantize import YoloV2Quantize
+from lmnet.networks.segmentation.lm_segnet_v0 import LmSegnetV0Quantize
+from lmnet.networks.segmentation.lm_segnet_v1 import LmSegnetV1Quantize
+from lmnet.quantizations import (
+    binary_mean_scaling_quantizer,
+    linear_mid_tread_half_quantizer,
+)
+
```

Comment:
Please change test python file name correspond to `QuantizeParamInit`'s file.

Reply:
I changed file's name to `test_quantize_param_init_arguments.py`